NOTICE.txt

```diff
@@ -1,2 +1,4 @@
-connectors-py
+Elastic Python connectors
```

Comment:
Not sure whether this is our official product name.

Reply:
This should be just the name of the repository, following our open-source guidelines (so `connectors-python`). I subtly remember this discussion with @gtback but don't remember where it was

Reply:
According to [the `NOTICE.txt` requirement for single-licence repo](https://github.com/elastic/open-source/blob/main/elastic-product-policy.md#for-repositories-holding-only-code-subject-to-the-elastic-license-egml-cpp), it should be the product name:
>{Product Name}
Copyright 2012-2018 Elasticsearch B.V.

One example: https://github.com/elastic/elasticsearch-ruby/blob/main/NOTICE

Reply:
Ha didn't know, makes sense!